pyscriptjs/tests/integration/test_01_basic.py

```diff
@@ -43,29 +43,29 @@ def test_python_exception(self):
         assert tb_lines[0] == "Traceback (most recent call last):"
         assert tb_lines[-1] == "Exception: this is an error"
 
-    def test_python_exception_in_event_handler(self):
-        self.pyscript_run(
-            """
-            <button py-click="onclick()">Click me</button>
-            <py-script>
-                def onclick():
-                    raise Exception("this is an error inside handler")
-            </py-script>
-        """
-        )
-
-        self.page.locator("button").click()
-
-        ## error in console
-        tb_lines = self.console.error.lines[-1].splitlines()
-        assert tb_lines[0] == "[pyexec] Python exception:"
-        assert tb_lines[1] == "Traceback (most recent call last):"
-        assert tb_lines[-1] == "Exception: this is an error inside handler"
-
-        ## error in DOM
-        tb_lines = self.page.locator(".py-error").inner_text().splitlines()
-        assert tb_lines[0] == "Traceback (most recent call last):"
-        assert tb_lines[-1] == "Exception: this is an error inside handler"
+    # def test_python_exception_in_event_handler(self):
```

Comment:
Why is this test commented out? @FabioRosado

Reply:
That was a left over from debugging apologies, just reverted the change 👍